cloudwatch-snowflake/variables.tf

```diff
@@ -0,0 +1,35 @@
+variable "fivetran_group_id" {
+  type        = string
+  description = "Also know as external_id. Understand the group concept here: https://fivetran.com/docs/getting-started/powered-by-fivetran#creategroupusingtheui"
```

Comment:
this link anchor is currently dead, it is missing _a_ in: 'create**~a~**group'
should be:
https://fivetran.com/docs/getting-started/powered-by-fivetran#createagroupusingtheui

Reply:
Thanks!